integrated_channels/blackboard/client.py

```diff
@@ -579,23 +583,28 @@ def delete_course_from_blackboard(self, bb_course_id, bb_content_id=None):
         resp = self._delete(delete_course_path)
         return resp
 
-    def create_integration_content_for_course(self, bb_course_id, channel_metadata_item):
+    def create_integration_content_for_course(self, bb_course_id, channel_metadata_item, update=False):
```

Comment:
if it's not much of an ask I would like to recommend separation of methods here between update and create, as in 

`create_integration_content_for_course`

`update_integration_content_for_course`

mainly because we are doing two different things entirely, and also doing that will allow us to avoid the multiple boolean checks and massaging of the error messages too such as in  L605 (I presume the message for update case needs to say accordingly). It will make for easier reasoning of the code and errors too I think

what say you?

Reply:
yeah, I'm happy to separate these out